### PR TITLE
Geneset search

### PR DESCRIPTION
**Pull request by airenzp**
This draft PR adds a function to render the geneset search discussed and creates a test to use it. It can be tested from http://localhost:3000/testrun.html?name=genesetEdit. I am opening it to share what I have so far and see if this is what we meant. Please check. I will continue to work on it until we think is ready.

### CODE REVIEW

File: client/dom/genesetSearch.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+export function initGenesetSearch({ holder, genome, callback, geneList, mode = 'mutation', vocabApi }) {
+	const div = holder.append('div').style('width', '50vw')
+	div
+		.style('border-style', 'solid')
+		.style('border-width', '2px')
+		.style('border-color', '#eee')
+		.style('padding', '5px')
+	const headerDiv = div.append('div')
+	const inputSearch = headerDiv.append('input').attr('placeholder', 'Search genes')
```
Comment (siosonel) [inline client/dom/genesetSearch.js:9]: @airenzp this should use `import { addGeneSearchbox } from '#dom/genesearch'`

File: client/dom/test/genesetSearch.spec.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+tape('\n', function(test) {
+	test.pass('-***- dom/geneDictionary -***-')
+	test.end()
+})
+
+tape('Search genes test', function(test) {
+	test.timeoutAfter(3000)
+
+	runTests()
+
```
Comment (siosonel) [inline client/dom/test/genesetSearch.spec.js:99]: does not need to use a `runTests()` wrapper function

**Comment by siosonel** [comment]
@airenzp I think it's a good start, just need to more closely match the button, input, layout as shown in the drawing.

**Comment by xzhou82** [comment]
let's name it `geneSetEdit` as the purpose is to create or edit a gene set

**Comment by airenzp** [comment]
Thanks for the comments. I will continue to work on it

**Pull request closed without merging**